packages/web/src/components/edit/fields/price-and-audience/PriceAndAudienceField.tsx

```diff
@@ -554,7 +586,57 @@ export const PriceAndAudienceField = (props: PriceAndAudienceFieldProps) => {
       }
       icon={<IconHidden />}
       initialValues={initialValues}
-      onSubmit={handleSubmit}
+      onSubmit={(values) => {
```

Comment:
this is nice. this logic might need to be shared between web/mobile?

Reply:
mobile does not have these fields yet. soon